cornedbeef/src/main/java/com/swiftkey/cornedbeef/HighlightCoachMark.java

```diff
@@ -14,9 +16,12 @@
  * @author lachie
  */
 public class HighlightCoachMark extends InternallyAnchoredCoachMark {
+    private View mView;
 
     protected HighlightCoachMark(HighlightCoachMarkBuilder builder) {
         super(builder);
+        ((GradientDrawable) mView.getBackground().mutate()).setStroke(
```

Comment:
Is there a reason these are gradients? Should we have a todo to clean them up if it's not necessary?

Reply:
The XML is:
```
<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle" >
   <solid android:color="@android:color/transparent" />
   <stroke android:width="@dimen/highlight_coach_mark_stroke_width" android:color="@color/default_colour"/>
   <corners android:radius="@dimen/coach_mark_border_radius"/>
</shape>
```
I tried with ShapeDrawables but I got a class cast exception because they were GradientDrawables.

Reply:
How confident are we that Android's implementation will stay the same? Might they become a different kind of drawable in the future (or on another device) and cause this to break?

Reply:
It worked from Jelly Bean (the lowest emulator I could easily create) to Pie. I've now tested in Android 10 and it also works well. However, I don't think there's any guarantee for the future.
Should I wrap this (and the equivalent part in the bubble coach mark) in a try-catch, just in case?

Reply:
Yeah I think that's our best bet. Then if it breaks whoever is using it will just see that it's not working and it should be clear for them how to fix it.

Reply:
Done